contrib/devtools/README.md

```diff
@@ -40,6 +40,12 @@ would be changed to:
 
 ```// Copyright (c) 2009-2015 The Bitcoin Core developers```
 
+gen-manpages.sh
+===============
+
+A small script to automatically create manpages in ../../doc/man by running the release binaries with the -help option.
+This requires help2man which can be found at: https://www.gnu.org/software/help2man/
```

Comment:
Why not use a makefile target instead? All of that info would be readily available there.